doc/changelog.rst

```diff
@@ -54,6 +56,7 @@ Detailed Notes
 
 .. _PR301: https://github.com/CrayLabs/SmartRedis/pull/301
 .. _PR300: https://github.com/CrayLabs/SmartRedis/pull/300
+.. _PR296: https://github.com/CrayLabs/SmartRedis/pull/296
```

Comment:
typo: Pull request 299?

Reply:
I promise I'll get the changelog sorted out before the final commit of these two PRs